index.rst

```diff
@@ -333,42 +309,42 @@ The following level of integration tests (on the NCSA-integration test stand) ar
 .. TODO::
     DM has developed a way to do this, we should explore if this solution works for this case as well.
     For test data used in unit tests DM uses git-lfs to store repositories that are set up as eups packages.
-    Another possible solution is Travis, which is useds to test the LSST EFD helper class.
+    Another possible solution is Travis, which is used to test the LSST EFD helper class.
```

Comment:
You may want to add Jenkins as a possible solution because it is pretty straightforward to create a Jenkins instance locally to run CI jobs like that.